### PR TITLE
Update flight calculator with real numbers

### PR DESCRIPTION
**Pull request by cmansel**
Update flight calculator with real numbers, add a cap to the per flight price of £25, add a disclaimer line below the calculator

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -246,6 +215,9 @@ class Calculator extends Component {
               </Flex>
             </Flex>
           </Flex>
+          <Text textAlign="left" mb={1} className={css({marginBottom: 0, paddingTop: 20, paddingLeft: 3, fontSize: 12, color: 'grey'})}>
```
Comment (maxmurdoch) [inline src/components/Calculator.js]: I think `SmallText` will look a bit better here

Comment (cmansel) [reply src/components/Calculator.js]: I tried using it, I wasn't able to change the fontsize

Comment (cmansel) [reply src/components/Calculator.js]: Without changing the fontSize for all the smallText components anyway, that might just be my unfamiliarity with react. Antton wanted the text to be the this size.

Comment (maxmurdoch) [reply src/components/Calculator.js]: OK cool. Well in that case you probably want to use `BodyText`. `Text` is a base component used underneath `SmallText` and `BodyText` and isn't (or at least shouldn't) be used anywhere, so if we use it here it'll be an outlier aesthetically and in the code.

Comment (cmansel) [reply src/components/Calculator.js]: ok

**Pull request merged**